pennylane/interfaces/tf.py

```diff
@@ -32,6 +31,40 @@
     from tensorflow import Variable  # pylint: disable=unused-import,ungrouped-imports
 
 
+def unflatten_tf(flat, model):
+    """Restores an arbitrary nested structure to a flattened TF tensor.
+
+    See also :func:`_unflatten`.
```

Comment:
```suggestion
    See also :func:`~.unflatten`.
```
(since the private `_unflatten` is not documented)